python/cudf/cudf/tests/dataset_synthesizer.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+# This module is for generating "synthetic" datasets. It has been designed
+# for testing filtered reading.
+#
+# The purpose of synthetic datasets is for ensuring that when
+# certain phenomena (e.g., cardinality) are exaggurated, we perform better
+# than cuIO and pyarrow. We want to be able to note in what cases do we
+# perform better.
+#
+# For general comparison, we should use "real" datasets. 
+
+DEBUG = False
```

Comment:
Should we control this via environment variable instead of having someone edit the source itself?

Reply:
I removed the debug print statements because in hindsight I don't think they are actually all that useful.

The debug statements were printing various characteristics of each column of generated data (cardinality, is it sorted, etc.) displayed in a table. That might useful for interactive use. But for this PR, `dataset_synthesizer` is just being used as a module for generating data, so I think it's fine to leave them out.